### PR TITLE
add blog tags and post discussing SBOMs

### PR DESCRIPTION
**Pull request by dholbach**
cc @pjbgf

### CODE REVIEW

**Comment by dholbach** [comment]
https://deploy-preview-752--fluxcd.netlify.app/blog/2022/02/security-the-value-of-sboms/

File: content/en/blog/2022-02-09-security-the-value-of-sboms/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  change, etc).
+- Automation allows you to send alerts, if a compliance issue is
+  detected, e.g. if the licensing of updated/replaced dependencies
+  changes.
+- Missing components or required build files. Incompatible licenses
+  and more.
+
+One example of automating all of this could be to store SBOMs in <https://grafeas.io/>. This way you could search across your Estate for:
+
+- Find all images that are built from a particular Github commit that is known to have introduced a security problem.
```
Comment (pjbgf) [inline content/en/blog/2022-02-09-security-the-value-of-sboms/index.md:120]: Without the "find all" from each example aligns better with the preceding statement:
```suggestion
- Images that are built from a particular Github commit that is known to have introduced a security problem.
```

File: content/en/blog/2022-02-09-security-the-value-of-sboms/index.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- Find all images that were built by a certain version of a certain builder when that builder is known to have been compromised.
+- Find all images in my project that are impacted by `CVE-1234`.
+
+For policy enforcement, combining SBOMs with grafeas and [kritis](https://github.com/grafeas/kritis) would enable auto-blocking of any application that is vulnerable to a specific CVE for example. This is the basis of <https://cloud.google.com/binary-authorization/>, which goes a step further from SBOM and also touches attestation.
```
Comment (pjbgf) [inline content/en/blog/2022-02-09-security-the-value-of-sboms/index.md:124]:
```suggestion
For policy enforcement, [kritis](https://github.com/grafeas/kritis) can be used to leverage the information provided by SBOMs inside Grafeas to enforce policies inside of a cluster, enabling auto-blocking of applications that are vulnerable to a specific CVE for example.
```

I am not sure we should mention attestation at this point in time. Potentially later on referring to some things we implement in the Flux project.

**Review by pjbgf — COMMENTED** [review]
Really nice one @dholbach! Just a few nits from my side.

**Comment by dholbach** [comment]
Thanks so much for your edits and feedback @stefanprodan and @pjbgf ! :feather:

File: content/en/blog/2022-02-09-security-the-value-of-sboms/index.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+[Syft](https://github.com/anchore/syft) in the
+[SPDX](https://spdx.dev/) format.
+
+![SPDX logo](featured-image.png)
+
+The `spdx.json` file is available for download on the GitHub release page
+e.g.:
+
+```shell
+curl -sL https://github.com/fluxcd/flux2/releases/download/v0.25.3/flux\_0.25.3\_sbom.spdx.json | jq
```
Comment (stefanprodan) [inline content/en/blog/2022-02-09-security-the-value-of-sboms/index.md:77]: Why the `\`? This works fine ` curl -sL https://github.com/fluxcd/flux2/releases/download/v0.25.3/flux_0.25.3_sbom.spdx.json | jq`

**Comment by dholbach** [comment]
Thanks @staceypotter for the review as well!

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**